sklearn/utils/extmath.py

```diff
@@ -696,9 +696,9 @@ def _batch_mean_variance_update(X, old_mean, old_variance, old_sample_count):
 
 def _deterministic_vector_sign_flip(u):
```

Comment:
@rvraghav93 why did you change this? This has been a while, I don't remember the discussion so well. This looks like it will change behavior of other estimators.